fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/indexes/VersionIndexTest.java

```diff
@@ -2300,6 +2331,50 @@ public void upgradeFormatVersions(int testFormatVersion, boolean splitLongRecord
         }
     }
 
+    @Test
+    void testScanVersionIndex() throws Exception {
```

Comment:
It feels like this test should either be `testScanRemoteFetchVersionIndex()` or should take the `IndexFetchMethod` as a parameter (and adjust its behavior accordingly).